pkg/helpers/helpers.go

```diff
@@ -42,3 +42,10 @@ func PollImmediate(interval time.Duration, timeout time.Duration, condition func
 
 	return errors.New("timed out waiting for the condition")
 }
+
+// Enumerated type representing the platform underlying the cluster-under-test
+const (
+	PLATFORM_AWS           string = "aws"
+	PLATFORM_GCP           string = "gcp"
+	PLATFORM_HOSTEDCLUSTER string = "hostedcluster"
```

Comment:
I think Go's style for consts is generally still camel case instead of the ALL_CAPS C-style, but no big deal for me

Reply:
Ahh thanks for the tip! I was going off of the IO library's standard for constants (e.g. `O_RDONLY`) but a quick Google search reveals that those were compatibility-related exceptions. My latest commit uses the proper camel-casing.